modules/services/xcape.nix

```
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.xcape;
+
+in
+
+{
+  meta.maintainers = [ maintainers.nickhu ];
+
+  options = {
+    services.xcape = {
+      enable = mkEnableOption "xcape";
+
+      package = mkOption {
+        default = pkgs.xcape;
+        defaultText = "pkgs.xcape";
+        type = types.package;
+        example = literalExample "pkgs.xcape";
+        description = "The package to use for the xcape binary.";
+      };
+
+      timeout = mkOption {
+        type = types.nullOr types.int;
+        default = null;
+        defaultText = "<timeout ms>";
+        example = 500;
+        description = "If you hold a key longer than this timeout, xcape will not generate a key event. Default is 500 ms.";
+      };
+
+      mapExpression = mkOption {
+        type = types.attrsOf types.str;
+        default = {};
+        example = { Shift_L = "Escape"; Control_L = "Control_L|O"; };
+        description = ''
+          The expression has the grammar 'ModKey=Key[|OtherKey][;NextExpression]'
```

Can use some docbook:
```suggestion
          The expression has the grammar
          <literal>ModKey=Key[|OtherKey][;NextExpression]</literal>
```